src/dataflow/src/render/sources.rs

```diff
@@ -482,7 +410,9 @@ where
                                         self.as_of_frontier.clone(),
                                         &mut linear_operators,
                                         key_arity,
-                                        src.bare_desc.typ().arity(),
+                                        //TODO(petrosagg): Envelopes need to be told how to access
+                                        // the fields they need from the previous stage
```

Comment:
what does this comment mean?

Reply:
That any information that has to do with the schema of the inputs of the envelopes needs to be calculated during planning and passed to the envelopes. In this PR the debezium projection structs serve these roles